### ISSUE TITLE
What option should I be using to force watched state to be pushed to Trakt?

### ISSUE THREAD

**Issue by OmgImAlexis** [trakt/Plex-Trakt-Scrobbler#397]
My internet was hit and miss for about 24 hours which meant your plugin didn't scrobble anything to my Trakt account, I've tried full, push all as well as push for tvshows and movies and till the shows I have marked as watched in Plex aren't being pushed to Trakt.

Am I missing something?

**Comment by fuzeman** [comment]
"Full" sync should bring everything up to date, maybe the media hasn't been matched in your Plex libraries yet?

Try run a "Refresh" (via Plex/Web) for the episodes/movies that aren't being updated on your Trakt.tv profile, wait for the items to be logged as `Finished processing ...` on the Plex/Web status page, and then try run another "Full" sync in the plugin.

**Comment by OmgImAlexis** [comment]
It's definitely matched as it's showing up on Trakt in my collection for both my account and the others I have linked in the plugin.

I've tried a refresh on the show and on individal episodes and then waited 10 - 20 mins to make sure it all finished. Then did a "full" sync and still nothing. 

Are there any logs I can check?

Could it possibly be that because I have it marked as watched before it's not pushing the new watched status?

**Comment by fuzeman** [comment]
> Could it possibly be that because I have it marked as watched before it's not pushing the new watched status?

So the episodes already have a play on your Trakt.tv profile?

It looks like syncs won't add *additional* plays to your profile, will need to do some testing to see if we can implement this safely. *(without any risk of duplicate history additions)*

**Comment by OmgImAlexis** [comment]
Yes I do, I didn't realise I had since the plugin didn't sync down that I already watched them even though I have it setup to do so.

Maybe add an option to add additional plays like how the scrobbler works?

**Comment by fuzeman** [comment]
I've determined after more research this can't be implemented as we have no way of reliably identifying which items have new plays.

The plugin currently can only mark items as watched in Plex Media Server at the current time, so any items that are added to your libraries which have already been marked as watched on your Trakt.tv profile will also look like "new" plays.

**Comment by OmgImAlexis** [comment]
Is there no way to view the logs, see that something wasn't scrobbled while the internet was down, etc. and then push a new play to trakt?

**Comment by fuzeman** [comment]
Retrying of failed *scrobbles* might be possible... will try look into that next week.

With the history de-duplication improvements to the Trakt.tv API we might be able to implement it fairly easily now. *(simple request retrying previously could lead to duplicate history additions)*

------------------

> Is there no way to view the logs, see that something wasn't scrobbled while the internet was down, etc. and then push a new play to trakt?

We can possibly check the plugin action history for your account and retry any scrobbles that have failed, will need to do some more research to confirm this.

This will only work for sessions/plays that have been stored in the plugin database, any plays before installing the plugin (or since resetting the plugin database) can't be recovered.
